pkg/otlp/metrics/metrics_remapping.go

```diff
@@ -165,6 +182,19 @@ func copyMetric(dest pmetric.MetricSlice, m pmetric.Metric, newname string, div
 				dp.SetDoubleValue(dp.DoubleValue() / div)
 			}
 		}
+		// attributes mapping
+		if fixed := attributesMapping.fixed; fixed != nil {
+			for k, v := range fixed {
+				dp.Attributes().PutStr(k, v)
+			}
+		}
+		if dynamic := attributesMapping.dynamic; dynamic != nil {
+			for old, new := range dynamic {
+				if v, ok := dp.Attributes().Get(old); ok {
+					dp.Attributes().PutStr(new, v.AsString())
```

Comment:
nit: We can delay mapping to string and instead do this to preserve the type:
```suggestion
					v.CopyTo(dp.Attributes().PutEmpty(new))
```

Reply:
Done 👍